src/lib/y2storage/free_disk_space.rb

```diff
@@ -34,6 +34,10 @@ class FreeDiskSpace
     #   @return [DiskSize]
```

Comment:
NP: I would take the opportunity to fix this documentation: it should be `@return [Partitionable]`.

Reply:
And also in the constructor of this class: `@param disk [Partitionable]`